src/components/generic/Popup/Popup.scss

```diff
@@ -0,0 +1,11 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-popup;
+
+.#{$C} {
+  position: fixed;
+  z-index: 1;
+  border-radius: 0.75rem;
+  box-shadow: var(--wxc-options-list--normal--box-shadow);
```

Comment:
```scss
--wxc-popup--box-shadow
--wxc-popup--background
--wxc-popup--border

Reply:
actually, just border and box-shadow should belong to the popup

background and color should belong to the options list (--wxc-options-list--background, without "normal" as the options list does not have states)

the padding should also belong to the options list